web/index.js

```diff
@@ -1 +1,5 @@
-import * as wasm from "yatima-web";
+//import * as wasm from "yatima-web";
+
+import("./pkg").then(module => {
+  module.run();
+});
```

Comment:
Are you using an older version of npm? Nix should provide you with the correct version. I have not had any problems with this.

Reply:
Inside nix-shell, what runs is the system npm (/usr/bin/npm), not from nix. Maybe #111 will add npm to nix-shell alongside nodejs?

But, you're right: import .. from .. should work, will fix it. I'm not entirely sure, but I think that the problem I had was that, by using WasmPackPlugin, the `"yatima-web": "file:./pkg"` line on package.json didn't work anymore.